services/src/main/scala/cromwell/services/metadata/impl/WriteMetadataActor.scala

```diff
@@ -9,15 +9,16 @@ import cromwell.services.metadata.MetadataService.{MetadataServiceAction, PutMet
 import cromwell.services.metadata.impl.WriteMetadataActor.{WriteMetadataActorData, WriteMetadataActorState}
 import org.slf4j.LoggerFactory
 
+import scala.concurrent.ExecutionContext
 import scala.concurrent.duration._
 import scala.util.{Failure, Success, Try}
 
-final case class WriteMetadataActor(batchRate: Int, flushRate: FiniteDuration)
+class WriteMetadataActor(batchRate: Int, flushRate: FiniteDuration)
```

Comment:
Right, we weren't using it like a `case class` and the `final` was getting in my way. Not to mention my own personal "more judicious use of case classes" crusade